code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -51,9 +51,9 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		if(reac_volume >= 5)
 			var/obj/item/book/affectedbook = O
 			affectedbook.dat = null
-			to_chat(usr, "<span class='notice'>Through thorough application, you wash away [affectedbook]'s writing.</span>")
+			affectedbook.visible_message("<span class='notice'>The [affectedbook]'s writing is washed away by [id]!</span>")
```

Comment:
Isn't "the" already included when you reference an object directly? Won't this format as "The the book's writing" etc.